package.json

```diff
@@ -111,7 +111,7 @@
   ],
   "author": "Red Hat",
   "license": "Apache-2.0",
-  "homepage": "/",
+  "homepage": "http://kiali.io",
```

Comment:
I think this setting is used by react. I'll check

Reply:
> By default, Create React App produces a build assuming your app is hosted at the server root.
To override this, specify the homepage in your package.json, for example:

https://github.com/facebook/create-react-app/blob/master/packages/react-scripts/template/README.md#building-for-relative-paths

Reply:
It's not the first time this param is touched, its worth to add a ""comment"" above it:
```js
"homepage-comment": [
  "By default, Create React App produces a build assuming your app is hosted at the server root.",
  "To override this, specify the homepage in your package.json:",
  "We don't want this, so don't touch homepage."
  "https://github.com/facebook/create-react-app/blob/master/packages/react-scripts/template/README.md#building-for-relative-paths"
]
```

Reply:
Yikes!
That's a different usage from what is documented at npm in https://docs.npmjs.com/files/package.json#homepage

>The url to the project homepage.
> Example:
>     "homepage": "https://github.com/owner/project#readme"

I will revert that and place the comment.

Reply:
I know, that's why i was suggesting a new entry in the package.json ("homepage-comment") and using multiple quotes when saying ""comment"" :)

That's a suggested way to comment stuff in the package.json

Reply:
Ok; comment added.